packages/ia-components/sandbox/pagination/pagination-with-flexbox.jsx

```diff
@@ -86,6 +105,54 @@ class Paginator extends Component {
     }
   }
 
+  /**
+   * Recalibrate dimensions of pagininator, taking into account what the closest page is
+   *
+   * @param { boolean } newItemInView - optional toggle
+   */
+  recalibrate(scrollItemIntoView = false) {
+    const findClosestThreshold = function findClosestThreshold(scrollItemIntoView) {
+      const { scrollThresholds } = this.state;
+      const { itemInViewClass } = this.props;
+      const viewport = this.Paginator.current;
+
+      // we only want pagination recalibration when it is single column
+      // and when it has more than 1 page
+      const numberOfPages = Object.keys(scrollThresholds);
+      const isColumn = viewport.className === 'flexbox-pagination column';
+      if (!isColumn && (numberOfPages.length <= 1)) return;
+
+      let itemToView = null;
+      if (scrollItemIntoView) {
+        // focus on item
+        itemToView = viewport.querySelector(itemInViewClass);
+        itemToView.focus();
+      }
+
+      const viewportFlush = scrollItemIntoView ? itemToView.offsetLeft : viewport.scrollLeft;
+      const pageToShow = findKey(
+        scrollThresholds,
+        th => viewportFlush >= th.low && viewportFlush <= th.high
```

Comment:
SUGGESTION: Expand abbreviation `th` so it's more obvious what the variable holds.